release/models/bfd/openconfig-bfd.yang

```diff
@@ -208,43 +215,7 @@ module openconfig-bfd {
         interface.";
     }
 
-    leaf desired-minimum-tx-interval {
-      type uint32;
-      units microseconds;
-      description
-        "The minimum interval between transmission of BFD control
-        packets that the operator desires. This value is advertised to
-        the peer, however the actual interval used is specified by
-        taking the maximum of desired-minimum-tx-interval and the
-        value of the remote required-minimum-receive interval value.
-
-        This value is specified as an integer number of microseconds.";
-    }
-
-    leaf required-minimum-receive {
-      type uint32;
-      units microseconds;
-      description
-        "The minimum interval between received BFD control packets that
-        this system should support. This value is advertised to the
-        remote peer to indicate the maximum frequency (i.e., minimum
-        inter-packet interval) between BFD control packets that is
-        acceptable to the local system.";
-    }
-
-    // rjs: Could have required-minimum-echo-receive here, but this is
```

Comment:
Echo state is present in RFC 9314.  Useful even if configuration state for it is not done.  Clearly that'd need to be split from the config grouping.

Reply:
Ideally this could be picked up as a new PR?   Getting the config stuff merged would be a good first step :)

Reply:
It seems required-minimum-echo-receive  is rarely if ever supported .  Seems ok for a separate PR to address this.

This change removes what is essentially a 'what-if' or TODO comment.    This seems ok as well, as it's not a critical operational issue (given lack of implementation?)